### ISSUE TITLE
Chrome password manager triggering while viewer property hidden is set to true

### ISSUE THREAD

**Issue by arturbiernadski** [stephanrauh/ngx-extended-pdf-viewer#277]
**Describe the bug**
If ngx-extended-pdf-viewer is used nested in form, when its hidden parameter is set to true, Chrome password manager is inserting login in field before viewer.
I've checked html code generated by using ngx-extended-pdf-viewer and there is input password which is probably the cause of this behavior.
```html
<ngx-extended-pdf-viewer ...>
    ...
    <input class="toolbarField" id="password" type="password">
    ...
</ngx-extended-pdf-viewer>
```

If hidden parameter is set to false or not used at all, everything works as intended.

**Version info**
- Version of ngx-extended-pdf-viewer: 3.2.2
- Version of the JavaScript files pdf.js, pdf-worker.js and viewer.js (currently: default files vs. version 2.2 preview files): 2.4.456-build-228a591c

**Desktop (please complete the following information):**
 - OS: Windows, Linux
 - Browser: Chrome
 - Version 81.0.4044.113 (Official Build) (64-bit)

**To Reproduce**
Example code listed below

1. Create a form with a text field on website to which you have saved credentials.
2. Use ngx-extended-pdf-viewer in this form. Place it after the text field.
3. When you set value of hidden property to true, you should see saved login inserted into the text field.

**Screenshots**
![image](https://user-images.githubusercontent.com/9093719/79971343-73bcca00-8494-11ea-8c34-010d05459aae.png)

**Code example**

`random.component.html`
```html
<form [formGroup]="randomForm">
  <input formControlName="randomName" />
  <div>
    <ngx-extended-pdf-viewer class="full-size" useBrowserLocale="true"
                             [hidden]="true"
                             [src]="uploadedFile"
                             [textLayer]="true">
    </ngx-extended-pdf-viewer>
  </div>
</form>
```
`random.component.html`
```typescript
import { Component } from '@angular/core'
import { FormControl, FormGroup } from '@angular/forms'

@Component({
  selector: 'app-random',
  templateUrl: './random.component.html',
  styleUrls: ['./random.component.scss']
})
export class RandomComponent {
  randomForm = new FormGroup({
    randomName: new FormControl(''),
  })
  uploadedFile: any

  constructor() { }
}
```

**Comment by stephanrauh** [comment]
Weird bug. I guess it's caused by the password dialog of the PDF viewer that allows you to open protected PDF files.

**Comment by stephanrauh** [comment]
BTW, what's the `[hidden]` property? I don't find it in the list of attributes I've implemented. Does it work nonethess? I'd expect a compiler error.

**Comment by arturbiernadski** [comment]
Hidden is HTMLElement property in Angular used to add [hidden attribute ](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden)to DOM element. We use it to hide ngx-extended-pdf-viewer whenever there is no pdf to view.

**Comment by stephanrauh** [comment]
I've managed to improve the situation. If you PDF file doesn't require a password, or if you pass it programmatically, the Chrome's password manager is not triggered. However, I don't have a clue how to prevent it from triggering randomly if you're opening a password protected PDF file:

![image](https://user-images.githubusercontent.com/3045767/80918720-bae56d80-8d66-11ea-9c4f-60a2220df1cb.png)

**Comment by stephanrauh** [comment]
I'm closing the ticket because the partial improvement has landed with version 3.2.5.
